spec/requests/swagger_docs/v5/partner_financials_spec.rb

```diff
@@ -39,219 +39,34 @@
                     },
                     capitals: { "$ref" => "#/components/schemas/Capitals" },
                     vehicles: {
                       type: :array,
                       description: "One or more vehicles' details",
-                      items: {
-                        type: :object,
-                        required: %i[value date_of_purchase],
-                        properties: {
-                          value: {
-                            "$ref" => "#/components/schemas/positive_currency",
-                            description: "Financial value of the vehicle",
-                          },
-                          loan_amount_outstanding: {
-                            "$ref" => "#/components/schemas/currency",
-                            description: "Amount remaining, if any, of a loan used to purchase the vehicle",
-                          },
-                          date_of_purchase: {
-                            type: :string,
-                            format: :date,
-                            description: "Date vehicle purchased by the applicant's partner",
-                          },
-                          in_regular_use: {
-                            type: :boolean,
-                            description: "Vehicle in regular use or not",
-                          },
-                          subject_matter_of_dispute: {
-                            type: :boolean,
-                            description: "Whether this vehicle is the subject of a dispute",
-                          },
-                        },
-                      },
+                      items: { "$ref" => "#/components/schemas/Vehicle" },
                     },
                     dependants: {
                       type: :array,
                       description: "One or more dependants details",
-                      items: {
-                        type: :object,
-                        required: %i[date_of_birth in_full_time_education relationship],
-                        properties: {
-                          date_of_birth: {
-                            type: :string,
-                            format: :date,
-                            example: "1992-07-22",
-                          },
-                          in_full_time_education: {
-                            type: :boolean,
-                            example: false,
-                            description: "Dependant is in full time education or not",
-                          },
-                          relationship: {
-                            type: :string,
-                            enum: Dependant.relationships.values,
-                            example: Dependant.relationships.values.first,
-                            description: "Dependant's relationship to the applicant's partner",
-                          },
-                          monthly_income: {
```

Comment:
@starswan Just want to make sure if its correct "$ref": "#/components/schemas/currency"  for 
- monthly_income
- assets_value

Reply:
sounds correct - they both sound like currency values?